From: Nobody GBP User Philippe Coval <nobody+gbp+philippe.coval@vates.tech>
Date: Tue, 31 Mar 2026 15:03:30 +0200
Subject: mdadm-udev

---
 udev-md-raid-assembly.rules | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/udev-md-raid-assembly.rules b/udev-md-raid-assembly.rules
index d4a7f0a..9f5ed43 100644
--- a/udev-md-raid-assembly.rules
+++ b/udev-md-raid-assembly.rules
@@ -5,6 +5,9 @@
 ENV{ANACONDA}=="?*", GOTO="md_inc_end"
 # assemble md arrays
 
+# Also don't process disks that are slated to be a multipath device
+ENV{DM_MULTIPATH_DEVICE_PATH}=="1", GOTO="md_inc_end"
+
 SUBSYSTEM!="block", GOTO="md_inc_end"
 
 # skip non-initialized devices
@@ -33,6 +36,13 @@ GOTO="md_inc_end"
 
 LABEL="md_inc"
 
+# Make sure we don't handle dm devices when some limits are set.
+# And linux_raid_member only be set when change/remove event happen.
+# So we don't need to consider add event here.
+KERNEL=="dm-*", ENV{DM_UDEV_RULES_VSN}!="?*", GOTO="md_inc_end"
+KERNEL=="dm-*", ENV{DM_UDEV_DISABLE_OTHER_RULES_FLAG}=="1", GOTO="md_inc_end"
+KERNEL=="dm-*", ENV{DM_SUSPENDED}=="1", GOTO="md_inc_end"
+
 # Bare disks are ready when add event happens, the raid can be assembled.
 ACTION=="change", KERNEL!="dm-*|md*", GOTO="md_inc_end"
 
